resolvers/index.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
+  'Project Group': {
+    read: (projectGroup, authedUser) => {
```

Comment:
presumably these will be updated? LOL

Reply:
yes! this is just 1 set of access resolvers haha. ideally this shouldn't be in this PR at all, but...